README.md

```diff
@@ -1186,3 +1187,25 @@ In addition to all arguments above, the following attributes are exported:
 - `private_key` - Automatically generated private key (base64 format) if `csr` was not provided.;
 
 [^ Back to all resources](#resources)
+
+### `spacelift_worker_pools` data source
+
+`spacelift_worker_pools` represents the worker pools assigned to the Spacelift account.
+
+#### Example usage
+
+```python
+data "spacelift_worker_pools" "worker-pools" {}
+```
+
+#### Argument reference
+
+No arguments are supported.
+
+#### Attributes reference
+
+The following attributes are exported:
+
+- `worker_pools` - List of [`spacelift_worker_pool` data sources](#spacelift_worker_pool-data-source);
```

Comment:
It's not a list of data sources but of objects with the shape identical to those returned by `spacelift_worker_pool`.

Reply:
@marcinwyszynski Should I show the attributes that a `worker_pool` object could have here? Or still link it?

Reply:
@marcinwyszynski seems to me that if I show the attributes here as well you would have to touch both of the section if the `worker_pool` data source changes. Linking seems better in that case.